### ISSUE TITLE
New fork `odolbeau/phone-number-bundle`

### ISSUE THREAD

**Issue by odolbeau** [misd-service-development/phone-number-bundle#217]
Hi all.

Given the lack of responsiveness of this bundle maintainer's, we (@Nek-, @Taluu & I) decided to create & maintain a new fork: [odolbeau/phone-number-bundle](https://github.com/odolbeau/phone-number-bundle).

This fork is **immediately available** and you'll find instructions to use it [in the README](https://github.com/odolbeau/phone-number-bundle#update-from-misdphone-number-bundle).

This new version fixes following issues / PR:
#170 #194 #196 #200 #202 #203 #204 #206 #207 #212 

**What have been done so far:**

* Remove symfony/templating
* Remove deprecated DIC paremeters & aliases
* Add Czech translations
* Add Turkish translations
* Add Ukrainian translations
* Drop support for Symfony < 3.4
* Drop support for PHP < 7.2
* Drop support for JMS Serializer
* Drop support for PHP templates

**Planned work for next release:**
* Create a more robust test suite
* Ensure we're fully compatible with symfony5 (which should be the case)
* Create a dedicated recipe

If you have any question, don't hesitate to ask! :+1:

**Comment by odolbeau** [comment]
Of course, If you're willing to help, don't hesitate to contact us, new maintainers are always welcome!

**Comment by odolbeau** [comment]
@thewilkybarkid Would you consider to abandon the package `misd/phone-number-bundle` in favor of `odolbeau/phone-number-bundle` on packagist please?
Don't hesitate to contact me if you want to discuss about this.
https://packagist.org/packages/misd/phone-number-bundle

**Comment by rh389** [comment]
I've been reluctant to say anything here because this was never my project - I only helped out a bit because I had access as a former colleague at MISD and I don't like to see a good project unmaintained. I haven't worked with PHP in several years and I'm in no position to put any more work in here. 

It's not really my place to appoint maintainers or to declare this abandoned... however... it's pretty obvious that @thewilkybarkid isn't involved any more (hope you're well C!), I'm not either, and @odolbeau's fork has now been active for some time and looks like a really well-maintained effort.

I'm therefore adding @odolbeau as a maintainer here to resolve this fork situation in whatever way he sees fit, if he wishes. Better late than never, I hope. All the best! 🤜🤛

**Comment by rh389** [comment]
Oh, and I should add my apologies to anyone on this repo who hasn't got a response from us, especially contributors - sincere thanks for your efforts and sorry if they went unmerged or unanswered.

**Comment by odolbeau** [comment]
Thanks @rh389.
I think the best solution is to abandon this package (directly on packagist) in favor of `odolbeau/phone-number-bundle`.
Unfortunately, I think only @thewilkybarkid can do this.

**Comment by rh389** [comment]
Is there any difference between that and [`composer.json#abandoned`](https://getcomposer.org/doc/04-schema.md#abandoned)?

**Comment by odolbeau** [comment]
Never heard about that before, thanks a lot. :)

**Comment by robhogan** [comment]
Done - good luck!
